apps/studio/src/components/Datatable/InfoCell.tsx

```diff
@@ -0,0 +1,39 @@
+import { memo } from "react"
+import NextLink from "next/link"
+import { Link, Text, VStack } from "@chakra-ui/react"
+
+export interface InfoCellProps {
+  caption?: string | null
+  subcaption?: string | null
+  href?: string
+}
+
+export const InfoCell = memo(
```

Comment:
wow TIL there's such a thing as `memo`

Reply:
suggest using friday learning to read [react docs](https://react.dev/reference/react/memo) hahah. The docs are really very good.